### DISCUSSION TITLE
containerdbg?

### DISCUSSION THREAD

**Discussion by pritidesai** [tektoncd/pipeline#6233] [Q&A]
Just came across a container debugging tool - https://github.com/google/containerdbg. This is just a few months old tool - first commit was in August, 2022.

I haven't played around with it but looks interesting. Was wondering if any of you have heard of it and may be something we can leverage in Tekton pipelines?

**Comment by ehmm** [comment]
Hi @pritidesai , I'm one of the developers working on containerdbg, what kind of integration were you thinking of?
